static/app/views/performance/browser/resources/settings.ts

```diff
@@ -1,7 +1,11 @@
 import {t} from 'sentry/locale';
 
-export const MODULE_TITLE = t('Resources');
+export const MODULE_TITLE = t('Assets');
 export const BASE_URL = 'browser/resources';
+export const DATA_TYPE = t('Asset'); // Name of the data shown (singular)
```

Comment:
Had trouble finding a decent name for this, but its a variable that represents the singular version of the type of data on the page. It maps to the span, i.e were showing `Asset` spans in this module